src/components/molecules/Band/Band.tsx

```diff
@@ -1,15 +1,53 @@
 import React, { FC } from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import Img from 'gatsby-image'
 
 import styles from './band.module.css'
 
 export type BandProps = {
-  bgColor?: 'default' | 'primary' | 'secondary'
+  bgColor?: 'default' | 'primary' | 'secondary' | 'waves'
+  size?: 'small' | 'medium' | 'large'
 }
 
-export const Band: FC<BandProps> = ({ bgColor = 'default', children }) => {
+interface BandData {
+  file: {
+    childImageSharp: {
+      fluid: []
+    }
+  }
+}
+
+export const Band: FC<BandProps> = ({
+  bgColor = 'default',
+  size = 'small',
+  children,
+}) => {
+  const data: BandData = useStaticQuery(graphql`
+    query HeadingQuery {
```

Comment:
In the future, we should look into fragments so that our queries can be reuseable / extended.